### PR TITLE
Add Solheim68

### PR DESCRIPTION
**Pull request by Maartenwut**
## Description

Adds the Solheim68, and open source replacement PCB for the va68m.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by fauxpark — APPROVED** [review]

File: keyboards/maartenwut/solheim68/solheim68.h [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+) \
+{ \
+    { k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0a, k0b, k0c, k0d, k0e, k0f }, \
+    { k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, k1c, XXX, k1e, k1f }, \
+    { k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b, k2c, k2d, XXX, XXX }, \
+    { k30, k31, k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b, k3c, XXX, k3e, XXX }, \
+    { k40, k41, k42, XXX, XXX, XXX, k46, XXX, XXX, XXX, k4a, k4b, k4c, k4d, k4e, k4f }  \
+}
+
+#define LAYOUT_68_ansi_split_shift( \
```
Comment (noroadsleft) [inline keyboards/maartenwut/solheim68/solheim68.h:74]:
```suggestion
#define LAYOUT_68_ansi_split_rshift( \
```

File: keyboards/maartenwut/solheim68/solheim68.h [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+) \
+{ \
+    { k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0a, k0b, k0c, k0d, k0e, k0f }, \
+    { k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, k1c, k1d, k1e, k1f }, \
+    { k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b, XXX, k2d, XXX, XXX }, \
+    { k30, XXX, k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b, k3c, k3d, k3e, XXX }, \
+    { k40, k41, k42, XXX, XXX, XXX, k46, XXX, XXX, XXX, k4a, k4b, k4c, k4d, k4e, k4f }  \
+}
+
+#define LAYOUT_68_iso_split_shift( \
```
Comment (noroadsleft) [inline keyboards/maartenwut/solheim68/solheim68.h:89]:
```suggestion
#define LAYOUT_68_iso_split_rshift( \
```

File: keyboards/maartenwut/solheim68/info.json [hunk 3]
```diff
@@ -0,0 +1,397 @@
...
+                {"x":3.75, "y":4, "w":6.25},
+                {"x":10, "y":4, "w":1.25},
+                {"x":11.25, "y":4, "w":1.25},
+                {"x":12.5, "y":4, "w":1.25},
+                {"x":14.25, "y":4},
+                {"x":15.25, "y":4},
+                {"x":16.25, "y":4}
+            ]
+        },
+        "LAYOUT_68_ansi_split_shift": {
```
Comment (noroadsleft) [inline keyboards/maartenwut/solheim68/info.json:241]:
```suggestion
        "LAYOUT_68_ansi_split_rshift": {
```

File: keyboards/maartenwut/solheim68/info.json [hunk 4]
```diff
@@ -0,0 +1,397 @@
...
+                {"x":3.75, "y":4, "w":6.25},
+                {"x":10, "y":4, "w":1.25},
+                {"x":11.25, "y":4, "w":1.25},
+                {"x":12.5, "y":4, "w":1.25},
+                {"x":14.25, "y":4},
+                {"x":15.25, "y":4},
+                {"x":16.25, "y":4}
+            ]
+        },
+        "LAYOUT_68_iso_split_shift": {
```
Comment (noroadsleft) [inline keyboards/maartenwut/solheim68/info.json:318]:
```suggestion
        "LAYOUT_68_iso_split_rshift": {
```

File: keyboards/maartenwut/solheim68/config.h [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+//   #define RGBLIGHT_EFFECT_RGB_TEST
+//   #define RGBLIGHT_EFFECT_ALTERNATING
+// /*== customize breathing effect ==*/
+//   /*==== (DEFAULT) use fixed table instead of exp() and sin() ====*/
+//   #define RGBLIGHT_BREATHE_TABLE_SIZE 256      // 256(default) or 128 or 64
+//   /*==== use exp() and sin() ====*/
+//   #define RGBLIGHT_EFFECT_BREATHE_CENTER 1.85  // 1 to 2.7
+//   #define RGBLIGHT_EFFECT_BREATHE_MAX    255   // 0 to 255
+// #endif
+
```
Comment (noroadsleft) [inline keyboards/maartenwut/solheim68/config.h:85]:
```suggestion
```

File: keyboards/maartenwut/solheim68/keymaps/default/readme.md [hunk 6]
```diff
@@ -0,0 +1 @@
+This is the default keymap as found on the Vortex Core, with some added buttons for RGB and backlight control.
```
Comment (noroadsleft) [inline keyboards/maartenwut/solheim68/keymaps/default/readme.md:1]: Vortex Core?

**Comment by noroadsleft** [comment]
Also, all the license headers say `Copyright 2020 Dekkers` – did you intend to only have your last name given on those?

**Comment by Maartenwut** [comment]
Yes.

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**